src/App.tsx

```diff
@@ -1,12 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
+import { GoodsList } from './components/GoodsList';
+import { Good } from './react-app-env';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+
+  const showAll = async () => {
+    setGoods(await getAll());
```

Comment:
Will be better to save the result of async function in some variable and then use this variable for sending data in setGoods method